src/utils/vm/mod.rs

```diff
@@ -1,5 +1,10 @@
 pub mod err_map;
 pub mod mem_regions;
 
-pub const STACK_SIZE: usize = 4096;
+/* Note: these contants are different for Firedancer vs Agave.
+For both, the virtual stack is 64x pages of 4kB each, with 4kB gaps.
+- Firedancer allocates 64 * (4+4)kB and wastes, i.e. gaps are phisical.
+- Agave allocates the 64 * 4kB, i.e. gaps are virtual and handled by vm_to_host(). */
+pub const STACK_SIZE: usize = 64 * 4_096;
+pub const STACK_GAP_SIZE: usize = 4_096;
```

Comment:
Agave uses the stack frame size as the gap size. Maybe we should reflect that relationship as such:
```suggestion
pub const STACK_GAP_SIZE: usize = STACK_SIZE;
```

Reply:
ignore this, didn't realize `STACK_SIZE` was changed to size of the full stack not frame